recipes/quasielasticbayes/conda_build_config.yaml

```diff
@@ -0,0 +1,3 @@
+fortran_compiler:
+  - m2w64-toolchain_win-64                 # [win]
+  - gfortran                               # [linux]
```

Comment:
Based on what you wrote here you need to pin these, no?

Reply:
Thanks, I think that would be a good idea